guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -522,21 +539,44 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
             }
         }
 
+        // If only Ctrl-Alt-End is pressed, and we have a one keysym from each
+        // group, and one key is being released, send Ctrl-Alt-Delete.
+        if (END_KEYS[keysym] &&
+            !_.isEmpty(_.pick(ALT_KEYS, currentKeysPressedKeys)) &&
+            !_.isEmpty(_.pick(CTRL_KEYS, currentKeysPressedKeys))
+        ) {
+                event.preventDefault();
+                delete keysCurrentlyPressed[keysym];
+                substituteKeysPressed[keysym] = DEL_KEY;
+                $rootScope.$broadcast('guacSyntheticKeydown', DEL_KEY);
```

Comment:
Out of curiosity, have you tried just using `$scope.$broadcast()`? There may not be a need to use `$rootScope`.

Reply:
Have not - will check and see what happens.

Reply:
Seems fine, change committed.